dashboard/modules/dashboard_sdk.py

```diff
@@ -348,6 +348,15 @@ def _upload_package(
                     include_parent_dir=include_parent_dir,
                     excludes=excludes,
                 )
+
+            package_size = package_file.stat().st_size
+            if package_size > 100 * 1024 * 1024:  # 100 MiB
+                logger.warning(
+                    f"The package {package_path} is very large. "
```

Comment:
Most of this text is fine, but in this specific case, I think the language should explicitly reflect that the package is not just large, but larger than the maximum

Reply:
Done, I want to highlight that the max unzipped size is 100 MiB, but here, the comparison is with the zipped size, so in cases when the unzipped size is > 100 MiB, but zipped is less than < 100 MiB, we won't see this warning..

By the way, thank you for reviewing :)